src/pages/resources/recipes/clipboard/index.md

```diff
@@ -0,0 +1,66 @@
+
+# Clipboard
+Read and write content from and to the clipboard.
+
+
+## System requirements
+Please make sure your local environment uses the following application versions before proceeding.
+- InDesign v18.5 or higher
+- UXP version v7.1 or higher
+- Manifest version v5 or higher
+
+
+<InlineAlert variant="info" slots="header, text1, text2"/>
+
+Scripts and plugins
+
+**In plugins**, you should seek permission for `clipboard` access in the manifest. <br></br> IMPORTANT: Read about the [manifest permissions](../../../plugins/concepts/manifest/#permissionsdefinition) before you proceed.
+
+**In scripts**, assume this value to be '' and ignore the manifest details in the following example.//TODO Get value for scripts and ignore the manifest details below.
+
+
+## Example
+
+<CodeBlock slots="heading, code" repeat="2" languages="JavaScript, JSON" />
+
+#### JavaScript
+```js
+async function foo() {
+    const clipboard = navigator.clipboard;
```

Comment:
@kareemmyk  What other APIs are there in 'navigator' ?

Reply:
@kareemmyk These are not documented in our API reference. Please create a ticket.